instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/ParentContextExtractor.java

```diff
@@ -43,7 +36,7 @@ static Context fromXRayHeader(String parentHeader) {
         .getTextMapPropagator()
         .extract(
             Context.current(),
-            Collections.singletonMap(AWS_TRACE_HEADER_PROPAGATOR_KEY, parentHeader),
+            Collections.singletonMap(AWS_TRACE_HEADER_PROPAGATOR_KEY.toLowerCase(), parentHeader),
```

Comment:
It would be nice to store the lowercased as a constant

Reply:
+1